### PR TITLE
Task responsive into Dev

### PR DESCRIPTION
**Pull request by karimkefi**
Media queries task added to the html and css for resizing of screen

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -287,6 +299,10 @@ aside {
         font-size: 0.8rem;
     }
 
+    .articleBoxSmall p {
+        visibility: hidden
```
Comment (ailindoyle) [inline style.css]: missing semicolon

File: style.css [hunk 2]
```diff
@@ -287,6 +299,10 @@ aside {
...
+    }
+
     .footerBoxSmall {
         margin:5px;
```
Comment (ailindoyle) [inline style.css]: missing space between colon and 5

File: style.css [hunk 3]
```diff
@@ -260,6 +261,17 @@ aside {
...
+        float: left;
+    }
+
+    .articleBoxMedium img {
+        padding: 0px;
+        width: 280px;
+    }
+
     .footerBoxMedium {
         margin:5px;
```
Comment (ailindoyle) [inline style.css]: missing space between colon and 5

**Review by ailindoyle — COMMENTED** [review]
changes look good - no obvious problems from what i can tell - a couple of formatting comments

File: style.css (hunk 1 above)
Comment (karimkefi) [reply style.css]: changed to "display: none;"

**Review by ailindoyle — APPROVED** [review]
Looks good - happy to approve!

Visibility: hidden and display: none are fairly similar - neither are seen by the user but the former leaves the box in the document flow and the latter removes it, so which one you use depends on what you're trying to achieve

**Pull request merged**